### ISSUE TITLE
Crash with Iris mod

### ISSUE THREAD

**Issue by SecretlyJealous** [alphaqu/DashLoader-Legacy#37]
Crash report: https://pastebin.com/S4UiFgyB
I'm flying around a build and certain chunks in that area refuse to be visible. When I middle click to copy the block then right click on the build the game just crashes.
Video of the crash: https://gfycat.com/serpentinepassionategarpike
The game keeps crashing anytime I join the server again, removing the dashloader mod was the only way to stop the crashes

**Comment by pluiedev** [comment]
Judging by the crash report alone, Iris isn't involved in the crash at all -- it looks like a compatibility issue between Dash and (Iris) Sodium. 
Can you remove Iris, use [the official Sodium 0.2.0](https://github.com/CaffeineMC/sodium-fabric/releases/tag/mc1.16.5-0.2.0) and try again? It's good to be sure if it's actually caused by the latest Sodium versions (cc @alphaqu)

**Comment by SecretlyJealous** [comment]
> Judging by the crash report alone, Iris isn't involved in the crash at all -- it looks like a compatibility issue between Dash and (Iris) Sodium.
> Can you remove Iris, use [the official Sodium 0.2.0](https://github.com/CaffeineMC/sodium-fabric/releases/tag/mc1.16.5-0.2.0) and try again? It's good to be sure if it's actually caused by the latest Sodium versions (cc @alphaqu)

Yep the issue is related to iris-sodium and dashloader, removing either iris-sodium or dashloader fixes the crashing. The official sodium 0.2.0 does not crash when used with dashloader.

**Comment by pluiedev** [comment]
Ah I see. @alphaqu might need to investigate this further

**Comment by theRookieCoder** [comment]
There is a possibility that this was fixed in Sodium 0.3. Iris no longer requires a custom fork of Sodium. Tested with Iris 1.1.2, and I cannot replicate!
